vendors/microchip/secure_elements/app/example_trust_chain_tool/README.md

```diff
@@ -25,47 +25,75 @@ PKI system) and an intermediate certificate authority that mirrors the signing
 
-### Create the Root CA
+In this flow, you will create a Root CA, a Signing CA, and finally a device certificate
+in what is known as a "chain of trust".
+
+Your Root CA will be self-signed at the top of your chain of trust.  
+
+Your Signing Certificate will be trusted by your Root CA, and will be used to issue your device certificate.
+
+Your Device Certificate will be unique per device, and is trusted by your Signing Certificate.
 
+### Create the Root CA
+First, create your Root CA.  
 ```
-> ca_create_root.py 
+>python ca_create_root.py 
 ```
 
 ### Create the Signing CA
+Next, create your Signing CA.  This CA will be registered with your AWS IoT account.  
+In order to prove your ownership of the Signing CA, obtain a registration code from AWS.
 
 ```
-> ca_create_signer.py <code_from_aws>
+aws iot get-registration-code
 ```
 
-code_from_aws is the registration code required to verify the private key used
-for the signing certificate. This is obtained where you upload the signer
-into the AWS IOT cacertificatehub. 
+This code will be used to generate a verification certificate, 
+as well as a Signing CA certificate & keys.
 
-1) In the AWS console navigate to AWS IOT -> Secure -> CAs
-2) Click the "Register" button and then the "Register CA" button
-3) Copy the registration code in "Step 2"
-4) Run ca_create_signer.py with the registration code
-5) Select the CA certificate (default is signer-ca.crt)
-6) Select the Verification certificate (default is verificationCert.crt)
-7) Select Activate CA certificate & Enable auto-registration
-8) Click the "Register CA certificate" button
-9) Make note of the certificate ARN hash for later (the first few digits)
+```
+>python ca_create_signer.py <code_from_aws>
+```
 
-The intermediate signing certificate is now active and allows for automatic
-registration of new devices when they are created (requires lambda functions).
-At this time we are not going to configure the lambda functions for automatic
-registration so we'll upload the devices certificate manually.
+Now register your CA certificate with AWS using the following command, passing in signer-ca.crt to
+register that certificate with AWS, and verficationCert.crt to prove that signer-ca.key belongs to you.
+
+```
+aws iot register-ca-certificate --ca-certificate file://signer-ca.crt --verification-cert file://verificationCert.crt
+```
+
+This API returns the certificate ID for your new CA certificate (this is the 64 charcter hex value that follows arn:aws:iot:<region>:<account>:cacert/<ca-certificate-id>).
+
+Your signing CA certificate is now "inactive", meaning that it cannot yet be used.  Activate it by 
+calling 
+
+```
+aws iot update-ca-certificate --certificate-id <ca-certificate-id> --new-status ACTIVE
+```
+where <ca-certificate-id> is the ID you got back when you registered your signing CA.
+
+Note that if you are using Just In Time Registration or Just In Time Provisioning, you will also need to enable auto-registration.
```

Comment:
We could avoid bringing the whole JITP/JITR can of worms into this document by instead saying something like, "For the purposes of this lab test, we recommend that you enable auto-registration for convenience."

Reply:
I have removed this entire step to avoid confusion.